LibreNMS/Util/ObjectCache.php

```diff
@@ -53,6 +53,7 @@ public static function applications()
                 ->groupBy('app_type', 'app_instance')
                 ->orderBy('app_type')
                 ->get()
+                ->sortBy('show_name', SORT_NATURAL|SORT_FLAG_CASE)
```

Comment:
Where do we need natural sort?

Reply:
tried it .. only sorts Case indepented alphabetic if both where in ...